redbot/cogs/trivia/data/lists/music.yaml

```diff
@@ -0,0 +1,542 @@
+AUTHOR: Lemon (inthedark.org#0666)
+Which famous group performed the first ever song on Top Of The Pops in 1964?:
+- The Rolling Stones
+Which band has released albums titled Word Gets Around, Just Enough Education To Perform and Pull The Pin?:
+- Stereophonics
+Which supermodel is seen pole dancing in the White Stripes video for the song I Just Don't Know What To Do With Myself?:
+- Kate Moss
+Which Beatle led the way across the zebra crossing on the Abbey Road album cover?:
+- John Lennon
+Which world famous musician was born Farrokh Bulsara in Zanzibar in 1946?:
+- Freddie Mercury
+Which group formed in 1977 and named themselves after their financial status at that time?:
+- Dire Straits
+What was the title of Bob the Builder's second UK number one hit single?:
+- Mambo No 5
+Which supermaket is mentioned in Chas and Dave's song Rabbit?:
+- Sainsbury's
+Which singer had hits with Rubber Ball and The Night Has A Thousand Eyes?:
+- Bobby Vee
+Which country did Celine Dion represent when singing in the Eurovision Song Contest in 1988?:
+- Switzerland
+What group with a palindromic name had a hit single that was also a palindrome?:
+- ABBA
+- ABBA (with SOS)
+Which artist released an album and movie under the same title of Get Rich Or Die Tryin'?:
+- 50 Cent
+Which US rapper died on September 13th 1996 after being shot in a drive-by shooting after watching Mike Tyson's comeback fight 7 days earlier?:
+- Tupac Shakur
+- 2Pac
+Who wrote the Sinead O'Connor hit Nothing Compares 2 U?:
+- Prince
+Which pop singer did Debbie Rowe marry?:
+- Michael Jackson
+The pop groups Ace Of Base and The Cardigans both hail from which country?:
+- Sweden
+In musical notes, which is bigger - a crotchet or a semibreve?:
+- Semibreve
+Which British female singer had a 2006 hit with 'Rehab'?:
+- Amy Winehouse
+Which Barbadian singer stayed ten weeks at number 1 with 'Umbrella'?:
+- Rihanna
+What nationality was Mozart?:
+- Austrian
+Which boy's name was the title of the debut album released by Amy Winehouse?:
+- Frank
+Which singer had a string of hits in the 1970s and is often referred to as the 'Queen of Disco'?:
+- Donna Summer
+Which group had a number one album in 1979 with Regatta De Blanc?:
+- The Police
+Who had a number one hit single in 1964 with 'Little Red Rooster'?:
+- The Rolling Stones
+In 1963, which band became the first to reach number one with their first three singles?:
+- Gerry and the Pacemakers
+By what name is Schubert's Symphony No. 8 better known?:
+- Unfinished Symphony
+By what name is singer Pauline Matthews better known?:
+- Kiki Dee
+What is the common link between The Dave Clark 5, Fat Larry's Band and Harold Melvin and the Blue Notes?:
+- All three are named after their drummer
```

Comment:
I feel like this question may be a bit too ambiguous for a trivia game because there's so many different ways people can phrase this answer, it's not like there's a succinct way to say this concept in one or two words... and as you have it now, people will have to say this answer exactly to be correct. What do you think?

Reply:
yeah you're right, this particular answer is not appropriate for a trivia game and can be said in alot of ways